### DISCUSSION TITLE
securly bypass patched

### DISCUSSION THREAD

**Discussion by dekid69** [3kh0/ext-remover#899] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
My school patched insecurly exploit about 2 months now.
the way my school patched it is that they installed securely by forcing installing directly to google. not from chromestore.

the extension id is: knneimmpeamikmjijmnhjmmobddleohj
version is: 2.98.61

**Comment by S-PScripts** [comment]
(necropost to answer question)

but what is your question? how to fix insecurly?
